src/main/java/com/redhat/rhjmc/containerjfr/platform/openshift/OpenShiftPlatformClient.java

```diff
@@ -99,14 +101,22 @@ private boolean isCompatiblePort(EndpointPort port) {
         return "jfr-jmx".equals(port.getName()) || 9091 == port.getPort();
     }
 
-    private List<ServiceRef> createServiceRefs(EndpointSubset subset, EndpointPort port) {
+    private List<ServiceRef> createServiceRefs(
```

Comment:
This method isn't static, so I don't think it needs the `logger` instance passed as a parameter, does it?

Reply:
No, you're right!